hack/make-rules/verify.sh

```diff
@@ -38,6 +38,7 @@ QUICK_PATTERNS+=(
   "verify-boilerplate.sh"
   "verify-godep-licenses.sh"
   "verify-gofmt.sh"
+  "verify-multiarch-test-images.sh"
```

Comment:
Already we have the script name by this name for checking the image names used in `test/e2e/*.go` files, not really sure whether we can place these in the same file!